plugin/src/main/scala/Giter8Plugin.scala

```diff
@@ -123,7 +123,9 @@ object Giter8Plugin extends sbt.AutoPlugin {
         } else Nil
 
         // copy test script or generate one
-        val script = new File(out, "test")
+        // the final script should always be called "test.script"
+        // no matter how it was originally called by user
+        val script = new File(out, "test.script")
```

Comment:
I originally thought this was a bug, but the final script is always called "test". 

Unfortunately that conflicts with Play applications using default Play layout.